integration/kubernetes/nginx.bats

```diff
@@ -18,14 +18,15 @@
 load "${BATS_TEST_DIRNAME}/lib.sh"
 
 setup() {
-	nginx_image="nginx"
-	busybox_image="busybox"
-	service_name="${nginx_image}-service"
+	image_registry="docker.io"
+	nginx_image="${image_registry}/nginx"
+	busybox_image="${image_registry}/busybox"
+	service_name="nginx-service"
 	export KUBECONFIG=/etc/kubernetes/admin.conf
 	master=$(hostname)
 	sudo -E kubectl taint nodes "$master" node-role.kubernetes.io/master:NoSchedule-
-	sudo -E crioctl image pull "$busybox_image"
-	sudo -E crioctl image pull "$nginx_image"
+	sudo -E crictl pull "$busybox_image"
```

Comment:
This should not be handled by k8s? (by calling  cri-o api)

Reply:
Yes, it can be handled by k8s, but as we have a timeout below, sometimes when running on a slow network, you may be able to see failures because the image couldnt be downloaded on time, that is why I pull them before launching the workload.

Reply:
Thanks for clarify, also would be good to add a comment inside.